src/main/bash/sdkman-install.sh

```diff
@@ -204,6 +204,12 @@ function __sdkman_checksum_zip() {
 		__sdkman_echo_yellow "Metadata file not found at '${headers_file}', skipping checksum..."
 		return
 	fi
+	
+	if [[ "$sdkman_checksum_skip" == 'true' ]]; then
```

Comment:
I prefer to flip the config around to the positive to align with our other configurations. How about `sdkman_checksum_enable`. I would expect the default to be `false` for the time being until a bunch of us thoroughly test this.